server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java

```diff
@@ -611,6 +614,14 @@ private String getHdfsDir() {
     return dataDirs;
   }
 
+  public boolean isEnableMetricService() {
```

Comment:
there is a field called `private boolean enableMetricsWebService = true;` You need to delete `enableMetricsWebService` and use your field.

Reply:
Hi, I will fix this